docs/rxjava3.md

```diff
@@ -0,0 +1,11 @@
+# Mavericks + RxJava3
+
+[//]: # (TODO refine the documentation)
```

Comment:
are you planning to address this?

Reply:
I am currently in the process of building an rxjava demo app, and as part of that, I had intended to create comprehensive documentation that includes various examples taken directly from the demo itself,  However, I apologize for not including it at this time.